templating/twig_extension_repository.rst

```diff
@@ -40,7 +40,7 @@ Then, define the extension class as a service and tag it with the
 
         # app/config/services.yml
         services:
-            twig_extension.intl:
+            twig_extensions.intl:
```

Comment:
I chose that because the repo is called "Twig Extension*s*". Thus I'd call a corresponding bundle "TwigExtensionsBundle", which would give this name. 

Or should we better recommend prefixing it with `app`?